docs/contents/users/install/kubeadm-onsite.md

```diff
@@ -0,0 +1,164 @@
+        sudo docker tag public.ecr.aws/eks-distro/etcd-io/etcd:v3.4.14-eks-1-18-1 public.ecr.aws/eks-distro/kubernetes/etcd:3.4.3-0
+
+1. Add the RPM repository to Google cloud RPM packages for Kubernetes by
+creating the following */etc/yum.repos.d/kubernetes.repo* file:
+
+        [kubernetes]
+        name=Kubernetes
+        baseurl=https://packages.cloud.google.com/yum/repos/kubernetes-el7-$basearch
+        enabled=1
+        gpgcheck=1
+        repo_gpgcheck=1
+        gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
+        exclude=kubelet kubeadm kubectl
+
+1. Install the required Kubernetes packages:
+
+        sudo yum install -y kubelet kubeadm kubectl --disableexcludes=kubernetes
+
+1. Create */etc/modules-load.d/k8.conf* so it appears as follows:
+
+        br_netfilter
+
+1. Create the */var/lib/kubelet directory*, then modify the */var/lib/kubelet/kubeadm-flags.env* file so it appears as shown below:
+
+        sudo mkdir -p /var/lib/kubelet
+        cat /var/lib/kubelet/kubeadm-flags.env
+        KUBELET_KUBEADM_ARGS="--cgroup-driver=systemd —network-plugin=cni —pod-infra-container-image=public.ecr.aws/eks-distro/kubernetes/pause:3.2"
+
+1. Get compatible binaries for *kubeadm*, *kubelet*, and *kubectl*.
+You can skip getting *kubectl* for now if you want to work with the
+cluster from your laptop:
+
+        cd /usr/bin
+        sudo rm kubelet kubeadm kubectl
+        sudo wget https://distro.eks.amazonaws.com/kubernetes-1-18/releases/1/artifacts/kubernetes/v1.18.9/bin/linux/amd64/kubelet; \
+        sudo wget https://distro.eks.amazonaws.com/kubernetes-1-18/releases/1/artifacts/kubernetes/v1.18.9/bin/linux/amd64/kubeadm; \
+        sudo wget https://distro.eks.amazonaws.com/kubernetes-1-18/releases/1/artifacts/kubernetes/v1.18.9/bin/linux/amd64/kubectl
+        chmod +x kubeadm kubectl kubelet
+
+1. To enable the *kubelet* service, type the following:
+
+        sudo systemctl enable kubelet
+
+1. Run the *kubeadm init* command, identifying the eks-distro image repository and Kubernetes version. Make any corrections requested for *kubeadm init* to complete successfully:
+
+        sudo kubeadm init --image-repository public.ecr.aws/eks-distro/kubernetes --kubernetes-version v1.18.9-eks-1-18-1
+        [init] Using Kubernetes version: v1.18.9-eks-1-18-1
+        [preflight] Running pre-flight checks
+        ...
+        [kubelet-finalize] Updating "/etc/kubernetes/kubelet.conf" to point to a rotatable kubelet client certificate and key
+        [addons] Applied essential addon: CoreDNS
+        [addons] Applied essential addon: kube-proxy
+        
+        Your Kubernetes control-plane has initialized successfully!
+
+    Your Kubernetes cluster should now be up and running. The kubeadm output should show the exact commands to use. If something goes wrong, correct the problem and run *kubeadm reset* to prepare you system to run *kubeadm init* again:
```

Comment:
> The kubeadm output should show the exact commands to use.

Should show the commands to use for what?

Reply:
Good point. I'll change the sentence to say: "The kubeadm output shows the exact commands to use to add nodes to the cluster."

Reply:
Done